shell.js

```diff
@@ -6,13 +6,7 @@
 // http://github.com/arturadib/shelljs
 //
 
-var commonPath = './src/common';
```

Comment:
This reverts #482, which I realized was not the best approach. We could leave #482 in, but it's just extra-complicated code once this gets merged, so I think it's best to revert.